src/main/java/com/pinterest/secor/transformer/MessageTransformers.java

```diff
@@ -0,0 +1,8 @@
+package com.pinterest.secor.transformer;
+
+
+public interface MessageTransformers {
+
+	public byte[] transform(byte[] message);
```

Comment:
Rather than introducing a new interface, should we consolidate the transformer concept with the message parser (i.e., add transform() to MessageParser)? The upside is less config params, the downside is less flexibility.